ups/python_moto.table

```diff
@@ -0,0 +1 @@
+setupRequired(python)
```

Comment:
Since we also have a boto stub package, we should probably require that as well to be explicit.